test_unstructured_ingest/unit/test_paths.py

```diff
@@ -127,3 +133,91 @@ def test_fsspec_folder_fails():
 
     assert output_filename == Path("/fake_file2.txt.json")
     assert download_filename == Path("/fake_file2.txt")
+
+
+def test_post_init_invalid_protocol():
```

Comment:
Are these changes relevant to this PR directly? Or just something added along the way?